app/code/Magebit/Faq1/view/frontend/templates/question-list.phtml

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+<?php $collection = $block->getQuestions(); ?>
+        <h1>Frequently Asked Questions</h1>
+        <?php foreach ($collection as $item) { ?>
```

Comment:
You dont need to declare variable if you only use it once: ($block->getQuestions() as $item)

Reply:
foreach ($collection as $item) {
Check the core foreach, dont use {, use:
and at the end use "<?php endforeach; ?>" not  <?php } ?>